brickstrap.sh

```diff
@@ -224,58 +326,28 @@ function create-conf() {
         EV3DEV_RASPBIAN_MIRROR="http://ev3dev.org/raspbian"
     fi
     info "Creating multistrap configuration file..."
-    debug "BOARDDIR: ${BOARDDIR}"
-    for f in ${BOARDDIR}/packages/*; do
-        # check that the package file hasn't been blacklisted
-        if echo "$BLACKLIST_PACKAGE_FILES" | grep -q $(basename "$f"); then
-            continue
-        fi
-        #
-        # Use read || [ -n "$line" ] to make sure the last line is also fed to the do-block.
-        # Otherwise packages may be omitted from files which lack a trailing newline,
-        # because of the non-zero exit code of read which would cause while to skip the do-block & terminate the loop.
-        #
-        while IFS='' read -r line || [ -n "$line" ]; do
-            case "$line" in
-            \#*|\;*) # permit comments: lines starting with # or ; are ignored.
-            ;;
-            *)
-                # avoid redundant spaces, i.e.  empty lines are ignored.
-                # also check that the package line hasn't been blacklisted
-                if [ -z "$line" ] || echo "$BLACKLIST_PACKAGES" | grep -q "$line"; then
-                    continue
-                else
-                    PACKAGES="${PACKAGES} $line"
-                fi
-            ;;
-            esac
-        done < "$f"
-    done
+    debug "BOARDDIR: $(br_project_dir)"
```

Comment:
I'd rather remove those old debug statements, I had simply updated them to the 'correct' value.